lib/index.js

```diff
@@ -1,13 +1,21 @@
-const error = res => res.status(403).send('Insufficient scope');
+const boom = require('boom');
 
-module.exports = expectedScopes => {
+module.exports = (expectedScopes, options) => {
   if (!Array.isArray(expectedScopes)) {
     throw new Error(
       'Parameter expectedScopes must be an array of strings representing the scopes for the endpoint(s)'
     );
   }
 
   return (req, res, next) => {
+    const error = res => {
+      const err_message = 'Insufficient scope';
+      if (options && options.failWithError) {
+        return next(boom.forbidden(err_message));
```

Comment:
I think I'd rather not add an extra dependency only for this error. We could just emulate what boom does in this case so we don't have to 'own' this dependency moving forward.

```suggestion
        return next({
          statusCode: 403,
          error: "Forbidden",
          message: err_message
        });
```

Reply:
Sounds good!